modules/compliance-evaluate-kubeletconfig-rules.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: CONCEPT
+[id="compliance-evaluate-kubeletconfig-rules"]
+= Evaluating `KubeletConfig` rules against default configuration values
+
+Kubernetes infrastructure may contain incomplete configuration files at run time, and nodes will assume default configuration values for missing configuration options. Some configuration options can be passed as command line arguments. As a result, the Compliance Operator cannot verify the configuration file on the node is complete or correct, as it may be missing options used in the rule checks.
```

Comment:
```suggestion
Kubernetes infrastructure may contain incomplete configuration files at run time, and nodes will assume default configuration values for missing configuration options. Some configuration options can be passed as command line arguments. As a result, the Compliance Operator cannot verify if the configuration file on the node is complete, as it may be missing options used in the rule checks.
```